src/HoneybadgerLaravel.php

```diff
@@ -24,7 +24,7 @@ public function make(array $config): Reporter
             'notifier' => [
                 'name' => 'Honeybadger Laravel',
                 'url' => 'https://github.com/honeybadger-io/honeybadger-laravel',
-                'version' => self::VERSION,
+                'version' => self::VERSION.' (PHP SDK: '.Honeybadger::VERSION.')',
```

Comment:
How about this slightly-less-verbose format? 

```suggestion
                'version' => self::VERSION.'/'.Honeybadger::VERSION,
```

Reply:
That works too, I'm just thinking of someone else in the future wondering what the second version number means. 😅

Reply:
Yeah, that's fair... in this case I'm voting for efficiency over clarity 😁

Reply:
Ah, okay then.